### PR TITLE
Action Template Updates

### PR DESCRIPTION
**Pull request by mmwtsn**
Handful of misc. updates in an unfortunately large pull request.

:ocean:

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/campaign/carousel.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    // Increment or decrement slide position based on user's request
+    direction === 'prev' ? decrementCounter() : incrementCounter();
+     
+    // Assign 'visible' class to the requested slide 
+    $('#slide' + counter).addClass('visible');
+  }
+
+  // Bind click event to carousel buttons
+  $buttons.click(function() {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/campaign/carousel.js:39]: This should be `$.on("click", ...)`.

File: lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss [hunk 2]
```diff
@@ -195,18 +195,27 @@
...
-    .tip-header {
+    .tip-header-wrapper {
       display: inline-block;
       text-decoration: none;
 
       a {
         text-decoration: none;
       }
+
+      span.bullet {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/feature/_campaign.scss:206]: Could probably do this with a pseudo-element to keep it out of the markup.

**Comment by DFurnes** [comment]
Looks great. :sun_with_face:

**Comment by aaronschachter** [comment]
Ran a new ds build --install and saw these errors:

```
js/campaign/carousel.js
  line 16  col 42  Expected an assignment or function call and instead saw an expression.
  line 23  col 42  Expected an assignment or function call and instead saw an expression.
  line 32  col 66  Expected an assignment or function call and instead saw an expression.

✖ 3 problems

```

Not sure if this is a bug or critical, but wanted to heads up

**Comment by mmwtsn** [comment]
@aaronschachter Thanks for catching that. Those notices [should be optional warnings](http://www.jshint.com/docs/options/#expr) as they can indicate a typo but aren't necessary a problem. We're OK with the lines in question and are [going to suppress the warning moving forward](https://github.com/DoSomething/dosomething/issues/818).

**Pull request merged**